controllers/rsync_common.go

```diff
@@ -39,6 +39,9 @@ import (
 const (
 	dataVolumeName = "data"
 	rcloneSecret   = "rclone-secret"
+	//nolint
```

Comment:
Is this needed?

Reply:
Yes otherwise golint checks read it as an exposed secret